package.json

```diff
@@ -37,15 +37,15 @@
   },
   "dependencies": {
     "@uswds/compile": "1.2.0",
-    "@uswds/uswds": "3.8.1",
-    "gulp": "^4.0.2",
+    "@uswds/uswds": "3.9.0",
+    "gulp": "^5.0.0",
```

Comment:
> [!TIP]
> This major Gulp update has caused issues with fonts and images. I tested `npm run copy-webfonts` and didn't see any issues.

Reply:
I took a look at your work in uswds/uswds-compile#96 to make sure the issue that was captured and resolved there wasn't going to be an issue here! I didn't find any issues either 👍